server/publications.js.coffee

```diff
@@ -67,6 +67,7 @@ Meteor.publish 'tickets', (selectedSquad) ->
         priority: issue.fields.priority.name
         labels: issue.fields.labels
         status: issue.fields.status.name
+        timespent: issue.fields.timespent/3600
```

Comment:
I'd like to see if we can use moment to do some nice formatting here. At the very least some rounding would be nice.